### PR TITLE
Bjk/fix helper

### PR DESCRIPTION
**Pull request by BJK**
make test rig fire change event on fillIn

### CODE REVIEW

File: test/lib/specHelperSpec.js [hunk 1]
```diff
@@ -30,26 +30,30 @@ var TestAction = assign(ActionPrototype, {
...
-    return {
-      value: 'initial value'
-    };
-  },
-  handleSubmit: function() {
-    var value = $(this.refs.value.getDOMNode()).val()
-    this.setState({value: value})
-  },
-  render: function() {
+class TestComponent  extends React.Component{
```
Comment (bartriordan) [inline test/lib/specHelperSpec.js]: @BJK Space before `{`

File: test/lib/specHelperSpec.js [hunk 2]
```diff
@@ -30,26 +30,30 @@ var TestAction = assign(ActionPrototype, {
...
+    //but this just shows something already set in state
+    this.setState({value: this.state.formVal})
+  }
+  render() {
     return (
       <div><h1>Test Component</h1>
         <div className='answer'>Form Value: {this.state.value}</div>
-        <input ref='value' defaultValue={this.state.value}/>
-        <a onClick={this.handleSubmit}>Submit</a>
+        <input ref='value' onChange={this.getVal.bind(this)} defaultValue={this.state.value}/>
```
Comment (bartriordan) [inline test/lib/specHelperSpec.js]: @BJK Space before `/>`

File: test/lib/specHelperSpec.js [hunk 3]
```diff
@@ -63,40 +67,25 @@ describe("Fluxxed up test helpers", function() {
...
-
-    beforeEach(()=> {
-      //todo either expose a method for this. OR -- use old school way.
-      var div = document.createElement('div');
-      node = React.render(<TestComponent/>, div)
-      domNode = $(node.getDOMNode())
+  describe("Test Rig", function() {
+    var rig = new TestRig()
+    beforeEach(() => {
+      rig.boltOn(<TestComponent/>)
```
Comment (bartriordan) [inline test/lib/specHelperSpec.js]: @BJK Space before `/>`

File: test/lib/specHelperSpec.js [hunk 4]
```diff
@@ -63,40 +67,25 @@ describe("Fluxxed up test helpers", function() {
...
-
-    afterEach(()=> {
-      React.unmountComponentAtNode(React.findDOMNode(node).parentNode);
+    afterEach(() => {
+      rig.boltOff()
     })
 
     it("renders", ()=> {
-      expect(domNode.find('h1').text()).to.equal('Test Component')
+      expect(rig.domNode.find('h1').text()).to.equal('Test Component')
```
Comment (bartriordan) [inline test/lib/specHelperSpec.js]: @BJK This works, as does `expect(rig.domNode.find('h1')).to.have.text('Test Component')`. Should we pick a style and stick with it?

**Comment by bartriordan** [comment]
@BJK Minor comments. Merge away once you update.

File: test/lib/specHelperSpec.js (hunk 4 above)
Comment (BJK) [inline test/lib/specHelperSpec.js]: I'm okay with there being a mix-- or if we insist, we can keep to the one I used. There's a lot of munging going on with these guys.. And aren't always using `.text()` to test equality. (So you see a lot of .to.equal block anyway).

**Pull request merged**